pageserver/src/tenant/timeline/detach_ancestor.rs

```diff
@@ -101,10 +115,65 @@ pub(super) async fn prepare(
         // a projection of the commited data.
         //
         // the error is wrong per openapi
-        return Err(NoAncestor);
+
+        {
```

Comment:
Is the TODO comment above still required? The new code seems to implement just that

Reply:
Yes most likely I missed TODO cleanup, thanks for pointing it out.

Reply:
TODO has been cleaned up.